github-runners/terraform/base/scripts/cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -eu
+
+echo "running clean up script to destroy current deployment environment."
+./config.sh remove --token ${GH_TOKEN}
+sudo rm -rf /home/ubuntu/actions-runner
+sudo rm -rf /home/ubuntu/_work
+pkill -9 -f actions
```

Comment:
Sholdn't action be stopped first before pruning the data ?

Reply:
`./config.sh remove` will handle that.